docs/ruleset_functions/get_baseline_system_types.md

```diff
@@ -0,0 +1,30 @@
+
+## get_baseline_system_types
+
+Description: Identify all the baseline system types modeled in a B-RMR.
+
+Inputs:  
+- **B-RMR**: The B-RMR that needs to get the list of all HVAC system types.
+
```

Comment:
I think this function will be more manageable if we instead use multiple functions, one for each baseline HVAC system type as shown below. The return values will be TRUE or FALSE. :
is_baseline_system_1(HVAC_id) 
is_baseline_system_2(HVAC_id)

Then "get_baseline_HVAC_system_types" would call these sub-functions to come up with the baseline system library.

Reply:
This note is added to the function for future development.

Reply:
Should the function return baseline HVAC system dictionary  like (System 1{HVACID1, HVACID2...), System 2 (HVACID2, HVACID3, ...)} etc.?